lib/server.js

```diff
@@ -38,9 +38,12 @@ function Server(universes, port) {
   this.universes = universes || [0x01];
   this.port = port || e131.DEFAULT_PORT;
   this._socket = dgram.createSocket('udp4');
-  this._lastSequenceNumber = 0;
-
+  this._lastSequenceNumber = {}
   var self = this;
+  self.universes.forEach(function (universe) {
+  	self._lastSequenceNumber[universe] = 0;
+	});
```

Comment:
Please check indentation. I use two spaces only (not tabs), also the `});` bit should be aligned to the beginning `self.universes.forEach(` part.